docs/content/preview/deploy/multi-dc/async-replication/async-transactional-setup-automatic.md

```diff
@@ -0,0 +1,74 @@
+---
+title: Set up transactional xCluster
+headerTitle: Set up transactional xCluster
+linkTitle: Setup
+description: Setting up transactional (active-active single-master) replication between two YB universes
+headContent: Set up transactional xCluster replication
+menu:
+  preview:
+    parent: async-replication-transactional
+    identifier: async-transactional-setup-1-automatic
+    weight: 10
+tags:
+  other: ysql
+type: docs
+---
+
+
+<ul class="nav nav-tabs-alt nav-tabs-yb">
+  <li >
+    <a href="../async-transactional-setup-automatic/" class="nav-link active">
+      Automatic
+    </a>
+  </li>
+  <li >
+    <a href="../async-transactional-setup-semi-automatic/" class="nav-link">
+      Semi-Automatic
+    </a>
+  </li>
+  <li >
+    <a href="../async-transactional-setup-manual/" class="nav-link">
+      Manual
+    </a>
+  </li>
+</ul>
+
+{{< note title="Note" >}}
+To use automatic transactional xCluster replication, both the Primary and Standby universes must be running v2.25.1 or later.
+{{< /note >}}
+
+
+Automatic transactional xCluster replication {{<tags/feature/tp>}} handles all aspects of replication for both data and schema changes.
+
+In particular, DDL changes made to the Primary universe are automatically replicated to the Standby universe.
+
+In this mode, xCluster replication operates at the YSQL database granularity. This means you only run xCluster management operations when adding and removing databases from replication, and not when tables in the databases are created or dropped.
+
+
+## Set up Automatic mode replication
+
+{{% readfile "includes/automatic-setup.md" %}}
+
+
+## Monitor replication
+For information on monitoring xCluster replication, refer to [Monitor xCluster](../../../../launch-and-manage/monitor-and-alert/xcluster-monitor/).
+
```

Comment:
I think a common question users will have is what is the progress of my DDLs, which I don't believe we currently have anywhere in the docs. 

The current way of doing this is via 
```
SET yb_xcluster_consistency_level = tablet;
SELECT * from yb_xcluster_ddl_replication.replicated_ddls order by ddl_end_time;
```

TBD on if we want to include that full command (or just the select statement) in the docs here? Or if we would rather include some function in the extension that executes that for the user instead?

Reply:
This should go under the Advanced monitoring, aka troubleshooting in https://support.yugabyte.com/hc/en-us/articles/29809348650381-How-to-troubleshoot-xCluster-replication-lag-and-errors?_gl=1*56mvcr*_gcl_au*MTI5NjQ0MDg1NS4xNzM3NjAzOTMx